openassessment/xblock/waffle_mixin.py

```diff
@@ -51,16 +53,27 @@ def _course_waffle_flag(flag_name):
         CourseWaffleFlag = import_course_waffle_flag()  # pylint: disable=invalid-name
         return CourseWaffleFlag(WAFFLE_NAMESPACE, flag_name)  # pylint: disable=feature-toggle-needs-doc
 
-    def team_submissions_enabled(self):
+    def is_flag_enabled(self, flag):
```

Comment:
Really minor nit: `is_feature_enabled` would be a better name, since it's answering a question about whether a flag or a switch is active.